### ISSUE TITLE
A/B Testing 

### ISSUE THREAD

**Issue by aaleksandraristic** [IanTDuncan/MealTime#27]
https://neilpatel.com/blog/how-to-market-your-mobile-app/

-Drop down, how many people are gonna use it if it's located on the top of the page or on the bottom of the page...

**Comment by BeepDroid** [comment]
Okay, so here are some samples of what an A and B version of a page may look like. The A version is the original mock-up from the storyboard, and the B version is a slightly altered version. 

The differences I made to the "Generate my meal" page is pushing some of the buttons closer together and higher up, making sure the dropdown is still included to illustrate that's still how those buttons will be operating. I've also added a sample meal picture to fill out the white space -- perhaps allowing some people to scroll through some more popular/common generations? If that's even possible. This is entirely design driven. 

The differences I've made to the meal page in and of itself is switching around the button strings, putting the grocery and servings options right under the meal instead of at the bottom of the page, and moving the button navigation between nutrition, recipe, and instructions at the very bottom. I've also moved around the meal name and the cost count, and removed the line seperation. 

![image](https://github.com/IanTDuncan/MealTime/assets/71675347/d251d54d-2c29-417d-8851-d339b4f0ac0c)
